content/renderer/media/media_permission_dispatcher.cc

```diff
@@ -142,15 +142,18 @@ void MediaPermissionDispatcher::OnPermissionStatus(
     blink::mojom::PermissionStatus status) {
   DVLOG(2) << __func__ << ": (" << request_id << ", " << status << ")";
   DCHECK(task_runner_->RunsTasksInCurrentSequence());
-
```

Comment:
Avoid deleting this line if it comes from upstream

Reply:
done.